src/notes-tree.ts

```diff
@@ -88,7 +102,7 @@ export class NotesTree implements vscode.TreeDataProvider<NoteItem> {
 	    let countPeding = 0;
 	    let countDone = 0;
 	    this.data = [];
-	    this.data = [new NoteItem('Pending'), new NoteItem('Done')];
+	    this.data = [new NoteItem('Pending', undefined, undefined, 'menu-pending'), new NoteItem('Done', undefined, undefined, 'menu-done')];
```

Comment:
For consistency, we are using context strings with a leading `$`:
```suggestion
	    this.data = [new NoteItem('Pending', undefined, undefined, '$menu-pending'), new NoteItem('Done', undefined, undefined, '$menu-done')];
```